### PR TITLE
Task/account page

### PR DESCRIPTION
**Pull request by giacomolicari**
Showing provider icon, address identicon and balance on header. Also fixed metamask loading issue.

### CODE REVIEW

File: src/components/WalletIntegrationProvider/index.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import { connectBlockchain, initGnosis } from 'actions/blockchain'
+// import { getCurrentAccount } from 'api'
```
Comment (andre-meyer) [inline src/components/WalletIntegrationProvider/index.js]: Remove please

File: src/components/WalletIntegrationProvider/index.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+export default class WalletIntegrationProvider extends Component {
+
+  constructor(props) {
+    super(props)
+    const { integrations, store } = props
+    const initializers = Object.keys(integrations).map(integrationName => integrations[integrationName])
+
+
+    // Execute providers inizialization sequentially
+    const init = (funcs, _store) => {
```
Comment (andre-meyer) [inline src/components/WalletIntegrationProvider/index.js]: I think the linter doesn't like `_variable` style variables. Please call it something else to convey more meaning

File: webpack.config.js [hunk 3]
```diff
@@ -16,28 +16,26 @@ const config = require('./src/config.json')
...
-      `${__dirname}/../gnosis.js/node_modules`,
+      'node_modules'
     ] },
   module: {
     rules: [
-      { test: /\.(js|jsx)$/, exclude: /(node_modules)/, use: 'babel-loader?babelrc=false&extends=' + path.join(__dirname, '/.babelrc') },
+      { test: /\.(js|jsx)$/, exclude: /(node_modules)/, use: 'babel-loader'},
       {
         test: /\.(jpe?g|png|svg)$/i,
         loader: 'file-loader?hash=sha512&digest=hex&name=img/[hash].[ext]',
```
Comment (andre-meyer) [inline webpack.config.js]: Can you revert your changes here? So we can merge better with #54

Comment (giacomolicari) [reply webpack.config.js]: Sure :)

File: src/components/WalletIntegrationProvider/index.js [hunk 4]
```diff
@@ -17,19 +16,19 @@ export default class WalletIntegrationProvider extends Component {
 
 
     // Execute providers inizialization sequentially
-    const init = (funcs, _store) => {
+    const init = (funcs, react_store) => {
       if (funcs.length > 0) {
-        return funcs[0].initialize(_store).then(
-          () => init(funcs.slice(1), _store),
+        return funcs[0].initialize(react_store).then(
+          () => init(funcs.slice(1), react_store),
```
Comment (andre-meyer) [inline src/components/WalletIntegrationProvider/index.js]: 😄 Sorry but can you please use camelCase? I'll merge asap then.

**Pull request merged**